lib/babe/verify_integration_test.go

```diff
@@ -145,7 +145,9 @@ func TestVerificationManager_VerifyBlock_Secondary(t *testing.T) {
 	secondaryDigest := createSecondaryVRFPreDigest(t, keyring.Alice().(*sr25519.Keypair),
 		0, uint64(0), uint64(0), Randomness{})
 	babeDigest := types.NewBabeDigest()
-	err = babeDigest.Set(secondaryDigest)
+	// NOTE: I think this was get encoded incorrectly before the VDT interface change.
+	// *types.BabeSecondaryVRFPreDigest was being passed in and encoded later
+	err = babeDigest.SetValue(*secondaryDigest)
```

Comment:
Does this PR change this behavior? I'm not seeing where

Reply:
After debugging this test I was getting a different error.  By passing in the non-pointer type on development, I got the same error, which leads me to believe that we shouldn't have been setting the VDT with pointer to secondary digest.